src/control/server/engine/config.go

```diff
@@ -1,5 +1,6 @@
 //
 // (C) Copyright 2019-2024 Intel Corporation.
+// (C) Copyright 2025 Hewlett Packard Enterprise Development LP
```

Comment:
Future note:   The (C) is not recognized as meaning anything at all in a copyright notice, so should not be present.
We should be removing the "(C)" from these files as we maintain them.

Copyright notices require one of, but should not have both, of either the word "copyright" or the special copyright symbol of an R inside a circle.   So even if the (C) was a legal substitute for the copyright symbol, it is still wrong if the word "copyright" is present.

Copyright notices are actually not even required to enforce copyright, it just provides better evidence.

This is a very common bug in software copyright messages.